ankihub/gui/overview.py

```diff
@@ -81,16 +87,54 @@ def _maybe_add_flashcard_selector_button() -> None:
     overview_web.eval(js)
 
 
+def _show_flashcard_selector_upsell_if_user_has_no_access(
+    on_done: Callable[[bool], None]
+) -> None:
+    user_details = AnkiHubClient().get_user_details()
+    has_access = user_details["is_premium"] or user_details["is_trialing"]
```

Comment:
We already have a method to check this information that was added on [PR#1055](https://github.com/AnkiHubSoftware/ankihub_addon/pull/1055)

```suggestion
    has_access = AnkiHubClient().is_premium_or_trialing()
```